src/expr/dtype.h

```diff
@@ -489,6 +499,47 @@ class DType
    * Helper for mkGroundTerm and mkGroundValue above.
    */
   Node mkGroundTermInternal(TypeNode t, bool isValue) const;
+  /**
+   * This method is used to get strict subfield types of this datatype.
+   *
+   * A subfield type T of a datatype type D is a type such that a value of
+   * type T may appear as a subterm of a value of type D.
+   *
+   * A *strict* subfield type T of a datatype type D is a type such that a
+   * value v of type T may appear as a subterm of a value of D, and moreover
+   * v occurs as a strict subterm of a non-datatype term in that value.
+   *
+   * For example, the strict subfield types of:
```

Comment:
```suggestion
   * For example, the strict subfield types of `T` in:
```
?

And simiarly for other examples?

Reply:
Yes, I'll change.